source/indexes/atlas-search-index.txt

```diff
@@ -20,26 +20,30 @@ Atlas Search Indexes
 Overview
 --------
 
-:atlas:`Atlas Search </atlas-search>` enables you to perform full-text searches on
-collections hosted on MongoDB Atlas. Atlas Search indexes specify the behavior of
-the search and which fields to index.
+In this guide, you can learn how to programmatically manage your Atlas
+Search and Atlas Vector Search indexes by using the {+driver-short+}.
```

Comment:
Aside: I see this `{+foo+}` syntax elsewhere. Is this an internal rST syntax extension?

Reply:
Yes, those are rST constants defined in the [snooty.toml](https://github.com/mongodb/docs-c/blob/master/snooty.toml) file of each repository (under "constants")